packages/management-local/gitops.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: management-local
+  namespace: kubezero
+spec:
+  description: Platform registry resources
+  clusterResourceWhitelist:
+    - group: '*'
+      kind: '*'
+  destinations:
+    - namespace: '*'
+      server: '*'
+  sourceRepos:
+    - https://github.com/DevOpsHiveHQ/kubezero
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: management-local
+  namespace: kubezero
+  annotations:
+    argocd.argoproj.io/sync-wave: '100'
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  project: management-local
+  source:
+    repoURL: https://github.com/DevOpsHiveHQ/kubezero
```

Comment:
DevOpsHiveHQ should be renamed "kubezero"

Reply:
For consistency, I will make it in the next PR :+1: